pxtblocks/blocklyloader.ts

```diff
@@ -1202,17 +1204,17 @@ namespace pxt.blocks {
     }
 
     function setBuiltinHelpInfo(block: any, id: string) {
-        const info = pxt.blocks.helpResources()[id];
+        const info = pxt.blocks.getBlockDefinition(id);
         setHelpResources(block, id, info.name, info.tooltip, info.url, getNamespaceColor(info.category));
     }
 
     function installBuiltinHelpInfo(id: string) {
-        const info = pxt.blocks.helpResources()[id];
-        installHelpResources(id, info.name, info.tooltip, info.url, getNamespaceColor(info.category))
+        const info = pxt.blocks.getBlockDefinition(id);
+        installHelpResources(id, info.name, info.tooltip, info.url, getNamespaceColor(info.category));
     }
 
     function setHelpResources(block: any, id: string, name: string, tooltip: any, url: string, colour: string) {
-        if (tooltip) block.setTooltip(tooltip);
+        if (tooltip && typeof tooltip === "string") block.setTooltip(tooltip);
```

Comment:
Blockly expects a string or a function that returns a string, so for our tooltips that are a string map, we should not pass those to Blockly (instead, we handle them in our code card)

I'll update this check to include functions